src/diagonal.works/b6/api/functions/sightline_test.go

```diff
@@ -36,20 +52,26 @@ func TestSightline(t *testing.T) {
 		name string
 		f    func(from s2.Point, radius s1.Angle, w b6.World) *s2.Polygon
 	}{
-		{"SightlineDefault", Sightline},
-		{"SightlineUsingPolygonIntersection", SightlineUsingPolygonIntersection},
-		{"SightlineUsingPolarCoordinates", SightlineUsingPolarCoordinates},
-		{"SightlineUsingPolarCoordinates2", SightlineUsingPolarCoordinates2},
+		{"TestSightlineDefault", Sightline},
```

Comment:
Nice work making the Test prefix consistant

Reply:
it s prob an illness but ty !!